backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -257,45 +260,77 @@ async def upload_boundaries_excel_sheet(
     try:
+        # Create input temporary file
         input_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
         content = await facility_with_supervisors.read()
         input_temp_file.write(content)
         input_temp_file.close()
         facility_with_supervisors_file_path = input_temp_file.name
 
+        # Create output file with timestamp
         timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
         output_filename = f"facility_with_supervisor_ingestion_results_{timestamp}.xlsx"
         output_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)